core/src/main/kotlin/org/evomaster/core/output/service/ApiTestCaseWriter.kt

```diff
@@ -62,6 +67,13 @@ abstract class ApiTestCaseWriter : TestCaseWriter() {
                 },
                 lines, insertionVars = insertionVars, skipFailure = config.skipFailureSQLInTestFile)
         }
+
+        var anyHostnameResolutionAction = false
+        if (initializingHostnameResolutionActions.isNotEmpty()) {
+            anyHostnameResolutionAction = handleHostnameResolutionActions(lines, initializingHostnameResolutionActions)
+        }
+
+        return anyHostnameResolutionAction
```

Comment:
why returning this?

Reply:
this is used to write, `clearDNSCache` at the end of the test case.